apps/gallery-new/utils/PresetUtils.ts

```diff
@@ -303,28 +303,14 @@ export const buttonShortcutOptions: ButtonShortcutVariant[] = ['accent', 'second
 
 export const buttonLinkOptions: ButtonLinkVariant[] = ['accent', 'secondary']
 
-export const transactionTypeOptions: TransactionType[] = [
-  'approve',
-  'bought',
-  'borrow',
-  'burn',
-  'cancel',
-  'claim',
-  'deploy',
-  'deposit',
-  'execute',
-  'mint',
-  'receive',
-  'repay',
-  'send',
-  'stake',
-  'trade',
-  'unstake',
-  'withdraw'
+export const transactionThumbnailOptions: TransactionThumbnailType[] = [
+  'token',
+  'nft',
+  'swap',
+  'fiat',
+  'unknown'
 ]
```

Comment:
The previous values are coming from the possible values of Zerion API, so we might want to keep the same array because we don't control this value manually on parent component